lib/oxidized/model/srlinux.rb

```diff
@@ -0,0 +1,51 @@
+class SRLinux < Oxidized::Model
+  using Refinements
+
+  # https://regex101.com/r/PGLSJJ/1
+  # prompt /^--{(\s\[[\w\s]+\]){0,5}[\+\*\s]{1,}running\s}--\[.+?\]--\s*\n[abcd]:\S+#\s*$/i
+  prompt /[ABCD]:\S+#\s*/
+
+  comment '# '
+
+  cmd :all do |cfg|
+    cfg.cut_both
+    # since multiline regexp for prompt doesn't seem to work, let's cut the remaining line with this
+    cfg.gsub! /--{(?:\s\[[\w\s]+\]){0,5}[\+\*\s]{1,}running\s}--\[.+?\]--/, ''
```

Comment:
The issue is vt100 control codes that get mixed in, causing the regex match to fail

See https://github.com/jbemmel/oxidized/blob/add-srlinux-model/lib/oxidized/model/srlinux.rb#L17 for a workaround